front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -203,13 +199,36 @@ const ImportTrainScheduleConfig = ({
         const formattedArrivalTime = `${startDate} ${arrivalTime}`;
         const formattedDepartureTime = `${startDate} ${departureTime}`;
 
+        let stopFor: number | undefined;
+
+        const arrivalDate = new Date(`${startDate}T${arrivalTime}`);
+        const departureDate = new Date(`${startDate}T${departureTime}`);
+        if (ocpType === 'stop') {
+          if (arrivalTime && departureTime) {
+            stopFor = Math.round((departureDate.getTime() - arrivalDate.getTime()) / 1000);
+          } else {
+            stopFor = 0;
+          }
+        } else if (ocpType === 'pass') {
+          if (arrivalTime && departureTime && arrivalTime !== departureTime) {
+            // Incorrect data, it shouldn't happen on a ocpType pass, skip step
+            return null;
+          }
```

Comment:
Given the way `arrivalTime` is defined above, I don't think this can ever happen? That is, if `ocpType === 'pass'`, then `arrivalTime` is always equal to `departureTime`?

Reply:
Indeed, we won't enter this condition, removing it